### ISSUE TITLE
Solaar is trying to interpret wrong reports

### ISSUE THREAD

**Issue by FFY00** [pwr-Solaar/Solaar#584]
When using a MX Master 3:

```
18:46:25,995    DEBUG [ReceiverListener:hidraw10] logitech_receiver.base: (14) => r[20 02 0200 0000F0FF00004F00000000]
18:46:25,995    DEBUG [ReceiverListener:hidraw10] logitech_receiver.base: sub_id = 2
18:46:25,995  WARNING [ReceiverListener:hidraw10] logitech_receiver.notifications: <PairedDevice(2,4082,MX Master 3)>: unrecognized Notification(2,02,00,0000F0FF00004F00000000) for feature DEVICE FW VERSION (index 02)
```

This report corresponds to mouse movement, it should not be interpreted by Solaar.

After a bit of debugging I found that `lib/logitech_receiver/base.py` passes this report as a "MEDIUM_MESSAGE". This is weird because I have never seen this anywhere and I don't remember the docs don't mentioning it. @Lekensteyn do you know where it comes from? Is this used by any device? Is it possible that @pwr made a mistake and thought this was a HID++ report type, even though is isn't?

**Comment by pfps** [comment]
These are DJ protocol messages.   They are documented in Unifying_receiver_DJ_collection_specification_draft.pdf
PR #646 should fix this problem.

**Comment by pfps** [comment]
Closed by PR #646
